src/plugins/presentation_util/public/components/solution_toolbar/items/quick_group.scss

```diff
@@ -0,0 +1,8 @@
+.quickButtonGroup {
+  .euiButtonGroup__buttons {
```

Comment:
Would it be possible to create/use a different class to avoid using EUI classes? Thanks!

Reply:
Actually, on second look, it appears that the related `border-radius` style might not even be necessary, as EUI is already giving it this styling. If I'm not mistaken, I think this selector and style can be removed altogether.